client/app/scripts/liveblog-edit/directives.js

```diff
@@ -534,15 +542,74 @@ define([
                 }
             };
         }])
-        .directive('freetypeLink', ['$compile', function($compile) {
+        .directive('freetypeText', ['$templateCache', function($templateCache) {
 
             return {
                 restrict: 'E',
-                template: '<input type="url" ng-model="link"/>',
-                controller: function() {
-                },
+                template: $templateCache.get('scripts/liveblog-edit/views/freetype-text.html'),
```

Comment:
Agreed, but a refactor here could take more time/efficiency  then it should. And porting the code to the new structure was done and should be easy to be updated.